smart_proxy_remote_execution_ssh.gemspec

```diff
@@ -31,4 +31,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency('rubocop', '0.32.1')
 
   gem.add_runtime_dependency('smart_proxy_dynflow', '>= 0.1.0', '< 0.3.0')
+  gem.add_runtime_dependency('net-ssh')
```

Comment:
I had to use version 4.2 of `net_ssh` for some compatibility reason... I can try to dog out the reason.

Reply:
I was testing with net-ssh 5.2.0 in my devel setup and everything seem to work (except Puma - I wonder if that could be related)